content/docs/1.7.0/advanced-resources/deploy/taint-toleration.md

```diff
@@ -70,7 +79,16 @@ You need to set tolerations for both types of components. See more details below
    To ensure that your preferred settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
 
 2. Set taint tolerations for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML to apply the values to all the components at once:
+     ```yaml
+       global:
+         tolerations:
+         - key: "key"
+           operator: "Equal"
+           value: "value"
+           effect: "NoSchedule"
+    ```
+   * You can also specify the tolerations for each user deployed components:
```

Comment:
```suggestion
   * You can also specify the tolerations for each system-managed component.
```

Reply:
its not "system-managed", it is still "user-deployed" here

The structure of the article is

- Install
    - user-deployed
    - system-managed
- upgrade
    - user-deployed
    - system-managed 

All I changed is only "user-deployed" in "install" and "upgrade" part